dev-docs/bidders/escalax.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: Escalax
+description: Prebid Escalax Bidder Adaptor
+biddercode: escalax
+usp_supported: true
+tcfeu_supported: true
+ccpa_supported: true
+coppa_supported: true
+schain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+pbjs: true
+pbs: true
+sidebarType: 1
+floors_supported: true
+prebid_member: false
+fpd_supported: false
+gvl_id: none
+multiformat_supported: will-bid-on-one
+ortb_blocking_supported: true
+userIds: all
+---
+
+### Note
+
+The Example Bidding adapter requires setup before beginning. Please contact us at <connect@escalax.io>
```

Comment:
```suggestion
The Escalax Bidding adapter requires setup before beginning. Please contact us at <connect@escalax.io>
```

I assume you don't mean the example bidding apdater :wink:

Reply:
oops, i didn't notice that😯